app/views/components/_figure.html.erb

```diff
@@ -1,12 +1,21 @@
 <%
   alt ||= ''
   caption ||= ''
+  credit ||= ''
 %>
 <figure class="app-c-figure">
   <img class="app-c-figure__image" src="<%= src %>" alt="<%= alt %>">
-  <% unless caption.blank? %>
     <figcaption class="app-c-figure__figcaption">
-      <%= caption %>
+      <% unless caption.blank? %>
+        <div class="app-c-figure__figcaption-text">
```

Comment:
Could we use paragraph tags here instead of divs? For both `app-c-figure__figcaption-text` and `app-c-figure__figcaption-credit`

Reply:
Sure! Not sure what the difference re: `<p>` and `<div>` is to be honest.

Reply:
Visually, they probably look the same unless you add your own styling. It's more about the semantics - a <p> (paragraph tag) is designed to wrap text, whereas a <div> is just a general containing element (which we tend to use for grouping or layout). Using semantic HTML means we are properly describing what the document/page is made up of: a load of `div` elements won't tell you a lot, but things like `p`, `image`, `a` etc give you more information.

Reply:
I hope that makes sense! 🙂